pkg/consts/consts.go

```diff
@@ -52,6 +52,9 @@ const (
 	// MaxConcurrencyLimits limits the max concurrency constraints for a specific function.
 	MaxConcurrencyLimits = 2
 
+	// MaxBatchTTL represents the maximum amount of duration the batch key will last
+	MaxBatchTTL = 10 * time.Minute
```

Comment:
The docs say the max batch timeout is 1 minute. Should this value match that?

Reply:
This is the redis TTL used for expiring the keys after batch started.